gems/aws-sdk-cloudfront/spec/cookie_signer_spec.rb

```diff
@@ -29,10 +29,7 @@ module CloudFront
                 }
               ]
           }
-          cookie = signer.signed_cookie(
-            "http://abc.cloudfront.net/images/image.jpg",
-            policy: policy.to_json
-          )
+          cookie = signer.signed_cookie(nil, policy: policy.to_json)
```

Comment:
could you add a separate test case instead of editing existing one?

Reply:
@cjyclaire Thanks for reviewing! Could you check it again?